src/plotman/configuration.py

```diff
@@ -46,6 +46,7 @@ class Archive:
     rsyncd_host: str
     rsyncd_user: str
     index: int = 0  # If not explicit, "index" will default to 0
+    mode: str = 'remote'
```

Comment:
So I think that maybe we can validate this here.

(imports would need to be added as well)
```suggestion
    mode: str = desert.field(
        default='remote',
        marshmallow_field=marshmallow.fields.String(
            validate=marshmallow.validate.OneOf(choices=['remote', 'local']),
        ),
    )
```

Probably don't bother for now, but having this be an enumeration might be good.  Making desert support `mode: typing.Literal['remote', 'local'] = 'remote'` would also be good.  But, oh well.  Another day.

Anyways, this can get us errors like the following at the time of load.

https://replit.com/@altendky/MicroSkeletalLinks-1
```python-traceback
Traceback (most recent call last):
  File "main.py", line 19, in <module>
    result = schema.loads('{"s": "blue"}')
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/marshmallow/schema.py", line 759, in loads
    return self.load(data, many=many, partial=partial, unknown=unknown)
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/marshmallow/schema.py", line 725, in load
    return self._do_load(
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/marshmallow/schema.py", line 907, in _do_load
    raise exc
marshmallow.exceptions.ValidationError: {'s': ['Must be one of: remote, local.']}
```